### PR TITLE
Feat: Manage_timelogs

### PR DESCRIPTION
**Pull request by dan-lv**
Feature for admin can create new timelogs, edit timelogs of user, see all timelogs of a user.

### CODE REVIEW

File: resources/views/admin/timelog/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        </form>
+    </div>
+    <div class="row trans-table">
+        <table class="table table-hover">
+            <thead class="thead-dark">
+                <tr>
+                    <th scope="col">Day</th>
+                    <th scope="col">Check In</th>
+                    <th scope="col">Check Out</th>
+                    <th scope="col">Name Staff</th>
```
Comment (cuongzinza) [inline resources/views/admin/timelog/index.blade.php]: Staff name

File: resources/views/admin/timelog/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                    <th scope="col">User ID</th>
```
Comment (cuongzinza) [inline resources/views/admin/timelog/index.blade.php]: Staff ID

File: resources/views/admin/timelog/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            </thead>
+            <tbody>
+                @foreach ($timeLogs as $timeLog)
+                <tr>
+                    <td>{{ $timeLog->day }}</td>
+                    <td>{{ $timeLog->check_in }}</td>
+                    <td>{{ $timeLog->check_out }}</td>
+                    <td>
+                        <a href="{{ route('manage.timelogs.timelog_user', $timeLog->user_id) }}">
+                            {{ $timeLog->user->name }}
```
Comment (cuongzinza) [inline resources/views/admin/timelog/index.blade.php]: link to user profile here, it does not make sense for linking to `manage.timelogs.timelog_user`

File: resources/views/admin/timelog/time_log_user.blade.php [hunk 4]
```diff
@@ -0,0 +1,37 @@
+@extends('layout')
```
Comment (cuongzinza) [inline resources/views/admin/timelog/time_log_user.blade.php]: filename: `timelog_user.blade.php`

File: resources/views/layout.blade.php [hunk 5]
```diff
@@ -19,7 +19,7 @@
             @yield('content')
         </div> 
 
-        <script src="{{ mix('js/library.js') }}"></script>
+        <script src="{{ mix('js/library.js') }}"></script>
```
Comment (cuongzinza) [inline resources/views/layout.blade.php]: remove trailing spaces

File: resources/views/admin/timelog/index.blade.php (hunk 3 above)
Comment (dan-lv) [reply resources/views/admin/timelog/index.blade.php]: I have fixed this

**Pull request merged**